src/pages/office/office.ts

```diff
@@ -63,12 +63,14 @@ import {
   getETAFaultText,
   getEco,
   getEcoFaultText,
+  getShowMeQuestionOptions,
+} from '../../modules/tests/test-data/test-data.selector';
+import {
+  getVehicleChecks,
   getSelectedTellMeQuestionText,
   getShowMeQuestion,
-  getVehicleChecks,
-  getShowMeQuestionOptions,
   getTellMeQuestion,
-} from '../../modules/tests/test-data/test-data.selector';
+} from '../../modules/tests/test-data/cat-b/test-data.cat-b.selector';
```

Comment:
Same here, office.ts is not cat-b specific. These shouldn't be cat-b specific selectors.
It might be the case that the office page has not been created as category specific yet. If that's true please add a comment so that we don't forget to change this in the future.

Reply:
Fair, will do